### DISCUSSION TITLE
impossible to delete a project

### DISCUSSION THREAD

**Discussion by baliaga31** [community/community#14637] [Projects and Issues] [answered]
I created a GitHub project (beta version). I closed it. I would like to delete it. I didn't find how to do? Any idea?

**Comment by EmanuelRodriguezBedeman** [comment]
> I created a GitHub project (beta version). I closed it. I would like to delete it. I didn't find how to do? Any idea?

I did the same exact thing, now I can't find any way to delete the project. Have you found anyway around?

**Reply by baliaga31** [reply]
I didn't find how to do.

**Comment by KnorpelSenf** [comment]
I created a project by accident, I did not even want to try out the feature yet. Now the project is there and cannot be deleted … what is this

**Comment by 2percentsilk** [comment, marked as answer]
We have recently shipped the ability to [delete projects](https://github.blog/changelog/2022-05-19-the-new-github-issues-may-19th-update/#%E2%9C%82%EF%B8%8F-delete-projects-you-no-longer-need). Give it a try and let us know how it goes!

**Reply by baliaga31** [reply]
That works ! Thank you :-)

**Reply by EmanuelRodriguezBedeman** [reply]
Thank you very much! Works like a charm!
